### DISCUSSION TITLE
SUM for more than 2 MPTT's solar energy  (GOODWE)

### DISCUSSION THREAD

**Discussion by tvarohp** [slipx06/sunsynk-power-flow-card#199] [Ideas]
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/122852911/51fbe327-1d2e-42b1-9d64-50cc4f45cf7e)
Could you please add possibility to have configurable also parameter for summ for solar generation_?
TX

**Comment by slipx06** [comment]
Hi. Can you explain what configurable options you are asking for?

**Comment by tvarohp** [comment]
I dont not see possibility to assign 3 entity to sollar production. Goodwe has entity such as MPTT1, MPTT2 and TOTAL. I would like to assign to entity as total (in picture is shown as 677W) this possiblity is not allowed in your setting, where you have only MPTT1 and MPTT2.
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/122852911/8e37f18b-8648-4869-8189-2fe2882c6e88)

**Comment by slipx06** [comment]
You can defined your own pv_total entity. Add the following top your config 

```
entities:
  pv_total: <your sensor here>
```

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/54b53bee-0c5b-4b40-81b2-f05ce674dfb2)

**Comment by tvarohp** [comment]
You are right. Sorry for that I did not see it
